test/Libraries/CoreNodesTests/WebTests.cs

```diff
@@ -28,5 +28,14 @@ public void WebRequest_EmptyUrl()
         {
             Assert.Throws<ArgumentException>(() => Web.WebRequestByUrl(""));
         }
+
+        [Test]
+        [Category("UnitTests")]
+        public void WebRequest_GitHubAPI()
+        {
+            string url = "https://api.github.com/repos/DynamoDS/Dynamo/issues?page=0&state=closed&per_page=10";
+            string result = Web.WebRequestByUrl(url);
+            Assert.IsNotNullOrEmpty(result);
```

Comment:
Please consider replacing the first test in this file, since this is a better version of it.

Reply:
> Please consider replacing the first test in this file, since this is a better version of it.

@mmisol It looks like the GitHub API has a rate limit of 60 requests per hour per originating IP for unauthenticated 
 requests (https://developer.github.com/v3/#rate-limiting). If this is exceeded then the test will likely fail due to a `403` response. I'm not familiar with how your test infrastructure is setup so is this likely to cause an issue?

Reply:
I've addressed this in commit 190a1e7. Removed the `WebRequest_ValidArgs` test and revised the GitHub API request to `/rate_limit` as requests to this endpoint do not count against your API rate limits.